src/Component/BlazorComponent/Components/Treeview/BTreeview.razor.cs

```diff
@@ -292,9 +294,9 @@ public async Task EmitSelectedAsync()
         {
             var selected = Nodes.Values.Where(r =>
             {
-                if (SelectionType == SelectionType.Independent)
+                if (SelectionType == SelectionType.Independent || SelectionType == SelectionType.Standalone)
```

Comment:
Here I Want To Make Standalone Same Independent But It Effect His Children But His Children Can't Effect Him